packages/vaul-vue/src/style.css

```diff
@@ -173,13 +173,6 @@
   }
 }
 
-@media (pointer: fine) {
-  [data-vaul-handle-hitarea]: {
```

Comment:
The `:` here caused a warning in the CLI. The CSS currently doesn't do anything because it is invalid, and even though it is included in the compiled CSS, it still contains the `:` in the compiled output and the styles therefore do not apply, so I deleted it.